src/pages/RealmScans/main.ts

```diff
@@ -0,0 +1,87 @@
+import { pageInterface } from '../pageInterface';
+
+export const RealmScans: pageInterface = {
+  name: 'RealmScans',
+  domain: 'https://realmscans.com/',
+  languages: ['English'],
+  type: 'manga',
+  isSyncPage(url) {
+    if (
+      j.$(
+        'div.entry-content.entry-content-single.maincontent > div.chnav.ctop.nomirror > span.selector.slc.l > div',
```

Comment:
The selectors everywhere are too, specific and would break with every small page change

Reply:
I shortened the selectors. Is this fine or should I try another approach?